src/templates/fiphr/import_bolus.json

```diff
@@ -1,14 +1,12 @@
 {
   "resourceType": "MedicationAdministration",
   "meta": {
-    "profile": [
-      "http://phr.kanta.fi/StructureDefinition/fiphr-sd-insulindosing-stu3"
-    ]
+    "profile": ["http://phr.kanta.fi/StructureDefinition/fiphr-sd-insulindosing-stu3"]
   },
   "language": "fi",
   "text": {
     "status": "generated",
-    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Koodi: Lyhytvaikutteinen insuliini (ins-short-fast)<br />Aika: {{formattedDate}}<br />Annos: {{insulin}} IU<br />Laite: {{deviceId}} (via {{_converter}})</div>"
+    "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\">Koodi: Lyhytvaikutteinen insuliini (ins-short-fast)<br />Aika: {{formattedDate}}<br />Annos: {{insulin}} iU<br />Laite: {{deviceId}} (via {{_converter}})</div>"

```

Comment:
Ei näin. IU on ihmiselle tutumpi termi. Siksi täällä narratiivissa yksikkönä IU.